conf/modules.config

```diff
@@ -44,6 +44,55 @@ process {
         ext.prefix = { "${meta.id}.fa" }
     }
 
+    withName: MMSEQS_DATABASES {
+        publishDir = [
+            path: { "${params.outdir}/databases/" },
+            mode: params.publish_dir_mode,
+            enabled: params.save_databases,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = [
+            params.taxonomy_mmseqs_databases_savetmp ? "" : "--remove-tmp-files" ,
+        ].join(' ').trim()
+    }
+
+    withName: MMSEQS_CREATEDB {
+        publishDir = [
+            path: { "${params.outdir}/taxonomy/mmseqs_createdb/" },
+            mode: params.publish_dir_mode,
+            enabled: params.taxonomy_mmseqs_save_intermedfiles,
```

Comment:
I wonder if this could also be consaidered `tmp` files, so could use the same parameter as above?

Reply:
good point